deploy/yaml/install.yaml

```diff
@@ -19552,7 +19577,11 @@ spec:
                   TargetDependents allows updating of dependent targets discovered but not
                   specified in the Target list
                 type: boolean
+              ttlAfterCompleted:
+                description: TTL for a completed update object.
+                type: string
               type:
+                description: Type of the update to perform.
```

Comment:
The description doesn't seem right?

I think it should include what format is expected, and how to disable GC (if possible).

Reply:
The descriptions are autogenerated from: https://github.com/pulumi/pulumi-kubernetes-operator/pull/810/files#diff-c203c1c46b5c663e63a1eeb95dacb02f899c2808c38f8b4645ee87bec34f6831.

In this case, `type` is actually a field describing what type of Update is being applied.